.github/workflows/check-generated-code-updates.yaml

```diff
@@ -0,0 +1,112 @@
+name: Check generated code is up-to-date
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+    branches:
+      - master
+
+jobs:
+  check_org_membership:
+    runs-on: ubuntu-latest
+    outputs:
+      isTeamMember: ${{ steps.teamAffiliation.outputs.isTeamMember }}
+    steps:
+      - name: Check user for team affiliation
+        uses: tspascoal/get-user-teams-membership@v3
+        id: teamAffiliation
+        with:
+          GITHUB_TOKEN: ${{ secrets.AUTO_BACKPORT_TOKEN }}
+          username: ${{ github.actor }}
+          team: 'dev'
```

Comment:
Why is the team hardcoded?

Reply:
it would be hard code (but maybe should be a secret)

it's one of few safe guards, so we can control who would be able to trigger this one

Reply:
Yeah, I was thinking about having it as a secret

Reply:
fixed it.